ethcore/src/lib.rs

```diff
@@ -115,10 +115,10 @@ extern crate kvdb_rocksdb;
 #[cfg(any(test, feature = "blooms-db"))]
 extern crate blooms_db;
 
-#[cfg(any(target_os = "linux", target_os = "macos", target_os = "windows", target_os = "android"))]
+#[cfg(any(target_os = "linux", target_os = "macos", target_os = "windows"))]
 extern crate hardware_wallet;
 
```

Comment:
we could eliminate those with `rust 2018` instead of both have those feature flags in both `Cargo.toml` and `lib.rs`, see https://blog.rust-lang.org/2018/12/06/Rust-1.31-and-rust-2018.html#cargo-features